packages/backend-platform/src/index.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Backend platform services for MetaMask.
+ */
+
+// Transaction and balance update types
+export type {
+  Transaction,
+  Asset,
+  Balance,
+  Transfer,
+  BalanceUpdate,
+  AccountActivityMessage,
+} from './types';
+
+// WebSocket Service - following MetaMask Data Services pattern
+export type {
+  WebSocketServiceOptions,
+  WebSocketMessage,
+  WebSocketConnectionInfo,
+  WebSocketSubscription,
+  InternalSubscription,
+  SubscriptionInfo,
+  WebSocketServiceActions,
+  WebSocketServiceInitAction,
+  WebSocketServiceConnectAction,
+  WebSocketServiceDisconnectAction,
+  WebSocketServiceSendMessageAction,
+  WebSocketServiceSendRequestAction,
+  WebSocketServiceGetConnectionInfoAction,
+  WebSocketServiceGetSubscriptionByChannelAction,
+  WebSocketServiceIsChannelSubscribedAction,
+  WebSocketServiceAllowedActions,
+  WebSocketServiceAllowedEvents,
+  WebSocketServiceMessenger,
+  WebSocketServiceEvents,
+  WebSocketServiceConnectionStateChangedEvent,
+  WebSocketState,
+  WebSocketEventType,
+} from './WebsocketService';
+export { WebSocketService } from './WebsocketService';
+
+// Account Activity Service
+export type {
+  AccountSubscription,
+  AccountActivityServiceOptions,
+  AccountActivityServiceSubscribeAccountsAction,
+  AccountActivityServiceUnsubscribeAccountsAction,
+  AccountActivityServiceActions,
+  AccountActivityServiceAllowedActions,
+  AccountActivityServiceAllowedEvents,
+  AccountActivityServiceAccountSubscribedEvent,
+  AccountActivityServiceAccountUnsubscribedEvent,
+  AccountActivityServiceTransactionUpdatedEvent,
+  AccountActivityServiceBalanceUpdatedEvent,
+  AccountActivityServiceSubscriptionErrorEvent,
+  AccountActivityServiceStatusChangedEvent,
+  AccountActivityServiceEvents,
+  AccountActivityServiceMessenger,
+} from './AccountActivityService';
+export {
+  ACCOUNT_ACTIVITY_SERVICE_ALLOWED_ACTIONS,
```

Comment:
Is there a reason why these constants are being exported? We don't typically do this in packages. If we need to use an action or event elsewhere we repeat it.